dart/packages/fury-core/lib/src/code_gen/analyse/impl/annotation/fury_class_annotation_analyzer.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:analyzer/dart/constant/value.dart';
+import 'package:analyzer/dart/element/element.dart';
+import 'package:fury_core/src/annotation/fury_class.dart';
+import 'package:fury_core/src/code_gen/analyse/analysis_type_identifier.dart';
+import 'package:fury_core/src/code_gen/entity/location_mark.dart';
+
+import '../../../../../fury_core.dart';
```

Comment:
Does dart support to use absulate path from project root path? Relative path is not easy to maintain and error-prune when refactoring code